nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-processors/src/main/java/org/apache/nifi/processors/cassandra/AbstractCassandraProcessor.java

```diff
@@ -132,17 +132,25 @@
             .addValidator(StandardValidators.CHARACTER_SET_VALIDATOR)
             .build();
 
+    // Relationships
     public static final Relationship REL_SUCCESS = new Relationship.Builder()
             .name("success")
-            .description("A FlowFile is transferred to this relationship if the operation completed successfully.")
+            .description("Successfully created FlowFile from CQL query result set.")
```

Comment:
This relationship is reused by PutCassandraQL as well, where there is no result set or query per se (it's a statement). That's why the doc is so generic. If you'd like to have the doc be more specific, you can create a REL_SUCCESS relationship in QueryCassandra using `new Relationship.Builder().from(AbstractCassandraProcessor.REL_SUCCESS).description("Your description override").build()`